osf/models/spam.py

```diff
@@ -193,18 +197,15 @@ def do_check_spam(self, author, author_email, content, request_headers, update=T
         remote_addr = request_headers['Remote-Addr']
         user_agent = request_headers.get('User-Agent')
         referer = request_headers.get('Referer')
-        try:
```

Comment:
What's the consequence of the AkismetClientError being uncaught? Will there be a better error shown in the admin app? Will it 500 and send to Sentry?

Reply:
It will be caught in the client, the current behavior is to fail silently for the end user, this at least allows the admin to see if an error happened.